### PR TITLE
Initial code for rendering the meta data

### PR DESCRIPTION
**Pull request by sv387**
Currently the implementation makes use of '$' as a delimiter to separate various components of metadata. Will change it to map with sentence delimiter.
Also need to work on reading the text comments entered by the user.

### CODE REVIEW

File: richreview/public/webapps/richreview/r2/model/doc_model.js [hunk 1]
```diff
@@ -1243,8 +1243,8 @@
         this.dom = document.createElement('div');
         this.dom.classList.toggle('r2_piecekeyboard', true);
         this.dom.classList.toggle('unselectable', true);
-        this.dom.setAttribute('aria-label', 'text comment');
-        this.dom.setAttribute('role', 'article');
+        this.dom.setAttribute('aria-label', 'This is a text comment');
```
Comment (DongwookYoon) [inline richreview/public/webapps/richreview/r2/model/doc_model.js]: Do we need 'this is a', which may can slow down the user's navigation.

File: richreview/public/webapps/richreview/r2/model/doc_model.js [hunk 2]
```diff
@@ -1243,8 +1243,8 @@
...
+        //this.dom.setAttribute('role', 'article');
```
Comment (DongwookYoon) [inline richreview/public/webapps/richreview/r2/model/doc_model.js]: Remove all the comments. In codes under collaboration, commented left-over codes only confuses others.

File: richreview/public/webapps/richreview/r2/model/dom_model.js [hunk 3]
```diff
@@ -49,9 +49,119 @@
 
         pub.remove = function(annot_id){
             var annot_id_esc = r2.util.escapeDomId(annot_id);
+            var parentPiece=$('#'+annot_id_esc).parent('.tc_piece');
```
Comment (DongwookYoon) [inline richreview/public/webapps/richreview/r2/model/dom_model.js]: Could you name variables with under-bar separation, and function names with Camel cases? That's the convention that I've used throughout this repository.

**Comment by sv387** [comment]
Closing this pull request to submit a new one for the metadata addition.

**Comment by DongwookYoon** [comment]
Hi Sahana,

Ok, make sure to fetch the new changes before creating PR.

Dongwook

On Fri, Dec 18, 2015 at 12:57 PM, Sahana Vijaykumar <
notifications@github.com> wrote:

> Closing this pull request to submit a new one for the metadata addition.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/DongwookYoon/RichReviewXBlock/pull/20#issuecomment-165856561>
> .
>



-- 
Best,
Dongwook


Dongwook Yoon
PhD Student
Information Science Department
Cornell University
T: +1-408-444-2877
E: dy252@cornell.edu

**Pull request closed without merging**